Dockerfile

```diff
@@ -22,6 +22,10 @@ RUN apk --update upgrade && \
     update-ca-certificates && \
     rm -rf /var/cache/apk/*
 
+# healthchecks: requires curl if not present in container
+# RUN apk --update --no-cache add curl
```

Comment:
Is curl included in image or not? Please either remove comment or install curl

Reply:
removed unnecessary comments